dataprovider/src/main/java/cm/aptoide/pt/dataprovider/model/v7/timeline/SocialCard.java

```diff
@@ -1,120 +0,0 @@
-package cm.aptoide.pt.dataprovider.model.v7.timeline;
-
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import java.util.List;
-
-/**
- * Created by jdandrade on 31/01/2017.
- */
-
-public abstract class SocialCard implements TimelineCard {
-
-  private final List<UserTimeline> likes;
-  private final List<CardComment> comments;
-  private final My my;
```

Comment:
This class should also be removed, since it doesn't have any usages

Reply:
delted